ethpm/assets/registry/registry_std_input.json

```diff
@@ -0,0 +1,30 @@
+{
+	"language": "Solidity",
+	"sources": {
+		"IndexedOrderedSetLib.sol": {
+			"urls": ["/Users/nickgheorghita/ethereum/py-ethpm/ethpm/assets/registry/contracts/IndexedOrderedSetLib.sol"]
+		},
+		"PackageDB.sol": {
+			"urls": ["/Users/nickgheorghita/ethereum/py-ethpm/ethpm/assets/registry/contracts/PackageDB.sol"]
+		},
+		"PackageRegistry.sol": {
+			"urls": ["/Users/nickgheorghita/ethereum/py-ethpm/ethpm/assets/registry/contracts/PackageRegistry.sol"]
+		},
+		"PackageRegistryInterface.sol": {
+			"urls": ["/Users/nickgheorghita/ethereum/py-ethpm/ethpm/assets/registry/contracts/PackageRegistryInterface.sol"]
+		},
+		"ReleaseDB.sol": {
+			"urls": ["/Users/nickgheorghita/ethereum/py-ethpm/ethpm/assets/registry/contracts/ReleaseDB.sol"]
+		},
+		"ReleaseValidator.sol": {
+			"urls": ["/Users/nickgheorghita/ethereum/py-ethpm/ethpm/assets/registry/contracts/ReleaseValidator.sol"]
```

Comment:
this file probably shouldn't be committed right **or** these should be updated to be relative URLs?

Reply:
ahh yup, that snuck through somehow